frontend/packages/client/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@creativebulma/bulma-tooltip": "^1.2.0",
     "@onflow/fcl": "^1.2.0",
+    "@onflow/sdk": "^1.1.1",
```

Comment:
Can you remove the package-lock.json file and get the yarn-lock.json file updated? `yarn install` on the frontend folder should do the trick